src/views/search/search.jsx

```diff
@@ -23,12 +25,14 @@ class Search extends React.Component {
         super(props);
         bindAll(this, [
             'getSearchState',
+            'handleChangeSortMode',
             'handleGetSearchMore',
             'getTab'
         ]);
         this.state = this.getSearchState();
         this.state.loaded = [];
         this.state.loadNumber = 16;
+        this.state.mode = '';
```

Comment:
This is correct - you can just use regular React state (rather than Redux state) to update the mode.